vagrant/roles/local.defaults/templates/config.yml.j2

```diff
@@ -27,6 +29,7 @@ config:
     {%- for idx in range(data.instances | default(1)) +%}
     {{ name }}{{ idx }}:
       groups: {{ data.groups }}
+      os: {{ data.os | default(environments[be].os | default(os)) }}
```

Comment:
Can we have the optional `os` parameter documented in _vagrant/roles/local.defaults/vars/main.yml_?

Reply:
Sure